app/assets/javascripts/miq_application.js

```diff
@@ -414,8 +414,6 @@ function miqSetButtons(count, button_div) {
   if (miqDomElementExists(button_div) && button_div.match('_buttons$')) { // Handle buttons that are not part of miq toolbars
     if (count === 0) {
       $('#' + button_div + ' button[id$=on_1]').prop('disabled', true);
-    } else if (count === 1) {
```

Comment:
looks like this was for `1` vs `1+`..

(but this *may* be dead code now)

Reply:
Introduced in https://github.com/ManageIQ/manageiq/pull/11952 . It seems to be for `1+` not just `1`. It can be tested in Configuration -> Settings -> select Region -> Red Hat Updates -> check one or more Appliances and see buttons Check for Updates/Register/Apply for ManageIQ updates.